docs/get_started/native_install/cross_tools.rst

```diff
@@ -64,9 +64,9 @@ https://launchpad.net/~team-gcc-arm-embedded/+archive/ubuntu/ppa.
 Installing the ARM Toolchain for Windows
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#.  Download and run the `installer <https://launchpad.net/gcc-arm-embedded/4.9/4.9-2015-q2-update/+download/gcc-arm-none-eabi-4_9-2015q2-20150609-win32.exe>`__
+#.  Download and run the `installer <https://developer.arm.com/-/media/Files/downloads/gnu-rm/5_4-2016q3/gcc-arm-none-eabi-5_4-2016q3-20160926-win32.exe?revision=b4aea310-4072-4388-8567-fc9cd4aded8b?product=GNU%20Arm%20Embedded%20Toolchain,32-bit,,Windows,5-2016-q3-update>`__
```

Comment:
`https://developer.arm.com/-/media/Files/downloads/gnu-rm/5_4-2016q3/gcc-arm-none-eabi-5_4-2016q3-20160926-win32.exe` should be enough for the URL, no?

Reply:
Btw, just checked ARM's website, latest version is:  [gcc-arm-none-eabi-8-2019-q3-update-win32.exe](https://developer.arm.com/-/media/Files/downloads/gnu-rm/8-2019q3/RC1.1/gcc-arm-none-eabi-8-2019-q3-update-win32.exe). This should probably be much better, if you are using Windows and don't mind giving it a bit of testing! :-P

Reply:
Or maybe [gcc-arm-none-eabi-8-2018-q4-major-win32.exe](https://developer.arm.com/-/media/Files/downloads/gnu-rm/8-2018q4/gcc-arm-none-eabi-8-2018-q4-major-win32.exe) to stay on the safe side...